### ISSUE TITLE
Unable to get GPS location on Android devices

### ISSUE THREAD

**Issue by insect1285** [Esri/Survey123Community#515]
When embedding a map into a form to capture location we cannot get a location from GPS on android devices.  The location icon appears in the status bar showing that the app is attempting to locate - it just never does.  The Windows application works fine.  Have not tested OSX.  Confirmed that devices support location and was working using Google Maps.

Map works fine for manual pin placement.

![screenshot_20160713-165938](https://cloud.githubusercontent.com/assets/4922876/16819644/6a1f6d30-491b-11e6-973c-351396bd030b.png)

**Comment by tbouckley** [comment]
Was an answer found to this? I am experiencing the same issue. Worked fine on OSX, just not android.

**Comment by tedrick** [comment]
@tbouckley - What android device are you using?

**Comment by tbouckley** [comment]
Hi Tedrick, I was attempting to run this on a Samsung Galaxy A3 and a oneplus one.

**Comment by raviwho** [comment]
Hi james,
we have noticed similar issue with our users. can you look into it??

**Comment by insect1285** [comment]
We found a work-around.  Seems that even if a device has GPS capability (thus Google Maps works), but is not connected to the internet, then Survey123 will not locate.  Once connected to the internet it works.  Our field staff are therefore tethering to an internet enabled device (cell phone) to get location.  

We are using Samsung Galaxy A3 (wifi only) tablets.

Suggested fix: Survey123 should use built-in GPS.

**Comment by tedrick** [comment]
@insect1285 - This was without on offline basemap, correct?  That issue will be addressed in the next release of Survey123

**Comment by insect1285** [comment]
@tedrick Correct.  They are using online-only.  Does this mean in the mean-time (prior to patch) that if the field staff download the map for offline use the GPS will work?
